examples/kandinsky2_2/text_to_image/train_text_to_image_decoder.py

```diff
@@ -896,7 +896,6 @@ def collate_fn(examples):
         images = []
         if args.validation_prompts is not None:
             logger.info("Running inference for collecting generated images...")
-            pipeline = pipeline.to(accelerator.device)
```

Comment:
Why is this going away?

Reply:
it called `enable_model_cpu_offload()` right after